Python/Github_Issues_PRs_Scrapper/README.md

```diff
@@ -0,0 +1,43 @@
+### For the Issues, it capture the following data points
+
+- Issue Creator
+- Issue Assignee
+- Corresponding PR (if any)
+- Issue Open/Close Date
+- People who commented on the Issue (Numbers and their Username)
+- Issue Content
+
+#### For the PRs capture the following data points
+
+- PR Creator
+- Merged/Closed
+- PR Open/Close/Merge Date
+- Reviewers (Numbers and their Username)
+- Who merged the PR?
+- PR Content
+- Files changed
```

Comment:
Can you please follow a standard template that we have on the home directory?

Reply:
@HarshCasper, you mean the rotten scripts readme, like heading, an introduction followed by the dependencies ..... etc ? will this order be okay?

Reply:
Follow [this](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md)

Reply:
oh okay 👍